org.kde.CrowTranslate.yaml

```diff
@@ -0,0 +1,73 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # The app supports capturing screen via KWin and Mutter DBus APIs, but
+  # access to it seems overly broad in spite of the app also supporting XDP API
+  #- --talk-name=org.kde.KWin
+  #- --talk-name=org.gnome.Shell
+rename-icon: crow-translate
+copy-icon: true
+command: crow
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+modules:
+  - name: crow-translate
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://invent.kde.org/office/crow-translate/.git
+        tag: 2.11.1
+        commit: 7113ff01a994584ed0a3ec557fc3a694fcb06b63
+        x-checker-data:
+          type: git
+    modules:
+      - name: tesseract
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - -DBUILD_SHARED_LIBS:BOOL=ON
+        cleanup:
+          - /bin
+        sources:
+          - type: archive
+            url: https://github.com/tesseract-ocr/tesseract/archive/refs/tags/5.4.1.tar.gz
+            sha256: c4bc2a81c12a472f445b7c2fb4705a08bd643ef467f51ec84f0e148bd368051b
+          - type: patch
+            paths:
+              - patches/tesseract/tesseract_neon.patch
+        modules:
+          - name: leptonica
+            buildsystem: cmake-ninja
+            builddir: true
+            config-opts:
+              - -DBUILD_SHARED_LIBS:BOOL=ON
+            sources:
+              - type: archive
+                url: https://github.com/DanBloomberg/leptonica/archive/refs/tags/1.84.1.tar.gz
+                sha256: ecd7a868403b3963c4e33623595d77f2c87667e2cfdd9b370f87729192061bef
+
+      - name: tessdata_fast
+        disabled: true
```

Comment:
If you are skipping this why is this added here?

Reply:
I just used what was added here: https://github.com/flathub/io.crow_translate.CrowTranslate/blob/ce7138a8686f7858ad2f99ce03ee11f5b8413f88/io.crow_translate.CrowTranslate.yaml#L70
Didn't even notice that it's disabled. Should I just remove it?

Reply:
I have no idea if you need it or not. Why was it added to the manifest and why was it disabled?